src/main/java/seedu/mtracker/ui/TextUi.java

```diff
@@ -44,6 +44,14 @@ public static void displayAddExpiryInstruction() {
         System.out.println("Expiry: ");
     }
 
+    public static void displayAddEntryPriceInstruction() {
+        System.out.println("Entry price: ");
+    }
+
+    public static void displayAddExitPriceInstruction() {
+        System.out.println("Exit price: ");
+    }
+

```

Comment:
If the parameters are optional, do add in the (optional) to it like remarks. It must have slipped my mind for Expiry, so feel free to add for that as well

Reply:
Nah, I think these should be compulsory.